### PR TITLE
Fix spelling of `Schrödinger` in tutorial

### PR DESCRIPTION
**Pull request by AbdullahKazi500**
### Summary
In the tutorial https://qiskit.org/ecosystem/nature/tutorials/01_electronic_structure.html Schroedinger was wrongly written
The correction of "schroedinger" to "Schrödinger" is important for several reasons. First and foremost, it is about giving proper recognition and respect to the scientist by using the correct spelling of his name. 
Furthermore, the correction highlights the significance of Erwin Schrödinger's work in quantum mechanics. His formulation of wave mechanics, which is described by the Schrödinger equation, provided a mathematical framework to describe the behavior of quantum systems. The Schrödinger equation is a fundamental equation in quantum mechanics that describes the time evolution of a quantum state.

University of Oxford https://www-thphys.physics.ox.ac.uk/people/SteveSimon/errors.html
Using an Umlaut over O is conventional and more appropriate 

### Details and comments
In summary, the correction from "schroedinger" to "Schrödinger" is justified based on proper naming conventions, respect for the individual, accuracy in scientific communication, recognition of historical and scientific significance

### CODE REVIEW

**Review by mrossinek — APPROVED** [review]
Thanks! :+1: 

Please sign the CLA so that we can merge this.

**Comment by AbdullahKazi500** [comment]
@mrossinek Done

**Comment by AbdullahKazi500** [comment]
I have signed CLA but I guess @mrossinek it is not showing up I don't know why

**Comment by mrossinek** [comment]
Earlier when I commented both comments were listed as `not signed`. So your signing definitely registered in one of them.

As to why there are two comments to begin with and one of them does not update is still a mystery to me. We will look into that separately though, regardless of this PR :+1:

**Comment by mrossinek** [comment]
Oh I just noticed now that you made this PR against the stable branch and not main.. Will port this to main manually now :+1:
